js/ViewPagerAndroid.android.js

```diff
@@ -12,7 +12,7 @@
 
 const React = require('react');
 const ReactNative = require('react-native');
-const {UIManager} = ReactNative;
+const {UIManager} = ReactNative.NativeModules;
```

Comment:
Why would you change the way we import it? I see in the docs how they:

```
import { UIManager } from 'react-native';
```

Reply:
Docs might not be in sync with actual code base. See for example:
https://github.com/facebook/react-native/blob/master/Libraries/ReactNative/UIManager.js#L19

Reply:
Yes you are right, should be
```
import { UIManager } from 'react-native';
```
will amend.

Reply:
@ferrannp amended.